lib/seagullService.js

```diff
@@ -40,7 +41,7 @@ function createServer(serverConfig, crudHandler, userApiClient, metrics) {
     return next(false);
   }
 
-  var seagullApi = require('./routes/seagullApi')(crudHandler, userApiClient, metrics);
+  var seagullApi = require('./routes/seagullApi')(crudHandler, userApiClient, metrics, gatekeeperClient);
```

Comment:
I see you are adding `gatekeeperClient` to the seagullApi, but I don't see that file in this PR.  Am I missing something?